sample-operators/webpage/k8s/crd.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This shouldn't be needed: the generator should be used for that and what we want people to use instead of manually writing CRDs.

Reply:
fixed. removed crd.yaml addd dependency.